build.sbt

```diff
@@ -1,16 +1,25 @@
 import bintray.Keys.{repository, bintrayOrganization, bintray}
-
-scalaJSSettings
-
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 organization               := "org.scalajs"
 
 name                       := "scala-parser-combinators"
 
-version                    := "1.0.2"
+version                    := "1.0.3-SNAPSHOT"
```

Comment:
Do not change the version number: we keep the version numbers of the upstream JVM version of the library.